workflow/README.md

```diff
@@ -0,0 +1,47 @@
+# TPS Pipeline
+
+**TPS workflow overview**
+
+![Alt Text](https://github.com/ajshedivy/tps/blob/visualization_v2/workflow/TPS_workflow.png)
```

Comment:
This can be a relative image path so that it works when merged.  You can also replace "Alt Text" with a description of the image.